app/views/articles/show.html.erb

```diff
@@ -1,12 +1,21 @@
 <p>
-	<strong>Title:</strong>
-	<%= @article.title %>
+  <strong>Title:</strong>
+  <%= @article.title %>
 </p>
 
 <p>
-	<strong>Text:</strong>
-	<%= @article.text %>
+  <strong>Text:</strong>
+  <%= @article.text %>
 </p>
 
+<h2>Comments</h2>
+<% @article.comments.each do |comment| %>
+  <%= render partial: "comments/comment", locals: { comment: comment } %>
```

Comment:
Add some CSS so that you can identify where a comment starts. If not, it seems like a list with a lot of comments with no commenters.